src/app/ipv/router.js

```diff
@@ -95,6 +94,6 @@ router.post(
   formRadioButtonChecked,
   handleJourneyAction,
 );
-router.get("/page/:pageId/back", handlePageBackButton);
-router.get("/*", updateJourneyState);
+
+router.get("/journey/:pageId/:action", updateJourneyState);
```

Comment:
(do you think this should be `/journey/:pageId` or `/page/:pageId`? This is fine though)

Reply:
Yeah I was torn between the two, but given we are technically firing a journey event I leaned towards journey (and the existing utilisation of the updateJourneyState route was using journey)

Reply:
Yeah, I can see the argument both ways